lib/omb-prompt-base.sh

```diff
@@ -85,6 +85,8 @@ CONDAENV_THEME_PROMPT_SUFFIX='|'
 PYTHON_THEME_PROMPT_PREFIX=' |'
 PYTHON_THEME_PROMPT_SUFFIX='|'
 
+OMB_PROMPT_SHOW_PYTHON_VENV=${OMB_PROMPT_SHOW_PYTHON_VENV:=true}
```

Comment:
I would like to make each theme selects the default. Currently, some themes show the information while others do not. I would like to leave the default behavior rather than make all the themes show the information. I wouldn't like to change the default of the themes, which is related to the reason that I mark similar PRs as "Customization Examples" and kept open.

You can put this line in the theme files that use the python venv information.

Reply:
I'll fix it.

Reply:
I have applied changes. Also, I added a commit 86ca670af7b550481ab8da2a90d34c6f5d8d8e5d that disables the python venv in `luan` by default, but I haven't yet actually determined which should be the default.

Reply:
Sounds good to me, makes sense since the themes are personalized. My thought process was to have a global setting, or rather something that could be shared between themes.